### DISCUSSION TITLE
想问下加水印的具体方法

### DISCUSSION THREAD

**Discussion by jtlin1** [Yuukiy/JavSP#185] [问答 | Q&A]
电脑小白 尝试了下有点不太明白怎么操作

**Comment by Yuukiy** [comment]
这个功能现在还不可用，之前有个临时版本运行时会报错

**Comment by jtlin1** [comment]
原来如此 我之前用了其他几个都没你这个好用 现在就差水印了加油

Yuukiy ***@***.***> 于 2024年1月5日周五 上午3:00写道：
